crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -41,17 +30,16 @@ async fn main() -> Result<()> {
 }
 
 async fn run() -> Result<()> {
-    let args = Cli::parse();
+    let conf = match config::get() {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            eprintln!("failed to read configuration: {e}");
```

Comment:
Return the result of `config::get` + some error context here and then let `main` a) report the error and b) call`std::process::exit`